app/ide-desktop/lib/dashboard/src/entrypoint.ts

```diff
@@ -27,4 +31,5 @@ main.run({
   projectManagerUrl: null,
   ydocUrl: null,
   appRunner: testAppRunner.TestAppRunner,
+  queryClient: commonQuery.createQueryClient(config => new reactQuery.QueryClient(config)),
```

Comment:
I think we don't need to construct the client that way because reactQuery.QueryClient is the same thing as queryCore.QueryClient: https://github.com/TanStack/query/blob/70c40e7876a2b1ab04e5128bb82a9110964a8f24/packages/react-query/src/index.ts#L4